src/flash.sol

```diff
@@ -23,9 +23,9 @@ contract DssFlash {
     // --- Data ---
     VatLike public  vat;    // CDP Engine
     address public  vow;    // Debt Engine
-    uint256 public  line;   // Debt Ceiling  [rad]
-    uint256 public  toll;   // Fee           [wad]
-    uint256 private locked; // reentrancy guard
+    uint256 public  line;   // Debt Ceiling     [rad]
+    uint256 public  toll;   // Fee              [wad]
+    uint256 private locked; // Reentrancy guard
 
     // --- Events ---
     event Rely(address indexed usr);
```

Comment:
We should remove `indexed` from the `Rely()` and `Deny()` events here. Using `indexed` will emit a hash of the address, which is helpful if you want to sort, group, or search these outputs, but you lose the data. The event should just be `event Rely(address usr);` in case we need to reconstruct the `ward` state later from events.

Debatable if we need it on the other events. It's probably fine on the `File()` events, and also probably `Mint()` since we can deduce that the `receiver` is the `msg.sender` of the transaction.

Reply:
On second look, the `receiver` is not necessarily the `msg.sender` here, so `Mint()` probably shouldn't be indexed, either.

Reply:
Are you sure about this? I've been treating it as the same as a database index that allows you to key on the value. IE "find me all the Rely/Denys for address X". As another example, DssGov is using indices on Rely/Deny: https://github.com/makerdao/dss-gov/blob/master/src/DssGov.sol#L110

Reply:
I dug back in to the docs and you're right, this will work for `address`.  ds-note uses `bytes` under the hood, which emits indexed data as a `keccak` hash rather than the value, so it's not recoverable. We've had some issues with that in dss and are moving away from it.  It looks like for datatypes [under 32 bytes](https://docs.soliditylang.org/en/develop/abi-spec.html#encoding-of-indexed-event-parameters) it stores the actual value, rather than a hash of the data, which means it should be fine here, I'll mention it to the team.